packages/sources/css/src/components/button/src/index.css

```
@@ -211,6 +200,7 @@
 
 .vtmn-btn--icon-alone span[class^='vtmx-'] {
   font-size: var(--vtmn-typo_text-2-font-size);
+  transform: translateY(rem(1px));
```

It feel nicer like this (IMHO)
```suggestion
  transform: translateY(rem(2px));
```